src/main/java/orlov/programming/spring_core_gym/dao/impl/user/DAOUsernameFindable.java

```diff
@@ -0,0 +1,7 @@
+package orlov.programming.spring_core_gym.dao.impl.user;
```

Comment:
https://google.github.io/styleguide/javaguide.html#s5.2.1-package-names
Package names use only lowercase letters and digits (no underscores). Consecutive words are simply concatenated together. For example, com.example.deepspace, not com.example.deepSpace or com.example.deep_space.